integration_tests/e2e/prisonerList/prisonerList.cy.ts

```diff
@@ -136,28 +134,38 @@ context(`Display the prisoner list screen`, () => {
       prisonerListPage //
         .hasResultsDisplayed(numberOfPrisonersNamedJohn)
     })
-
-    it('should clear filters to reset the search', () => {
-      // Given
-      cy.signIn()
-      cy.visit('/')
-      const prisonerListPage = Page.verifyOnPage(PrisonerListPage)
-      prisonerListPage //
-        .setNameFilter('some non existent search term')
-        .setStatusFilter('NEEDS_PLAN')
-        .applyFilters()
-        .hasNoResultsDisplayed()
-
-      const expectedResultCount = prisonerSearchSummaries.length
-
-      // When
-      prisonerListPage.clearFilters()
-
-      // Then
-      prisonerListPage //
-        .hasResultsDisplayed(expectedResultCount)
-        .hasNoSearchTerm()
-        .hasNoStatusFilter()
+    ;['VIEW', 'CONTRIBUTOR', 'MANAGER'].forEach(authority => {
```

Comment:
This test is a rework of the previous version, but iterates through 3 different user types (read only users, contributors, and managers)
For each of them the test gets to the prisoner-list page, applies a filter, and then clicks the "Clear" filter link.
The expectation is that for all user types they all land back on the prisoner-list page with the filters removed and a full set of results displayed.